channels-rke2.yaml

```diff
@@ -234,6 +234,15 @@ releases:
       harvester-csi-driver:
         repo: rancher-rke2-charts
         version: 0.1.400
+  - version: v1.21.7-rc2+rke3r3
```

Comment:
1. rke3 looks like a typo, should be rke2r3? 
2. We already have  entry for `v1.21.7` in this file -`v1.21.7+rke2r2` which hasn't been released. So I think we can replace it. Or do we need both r2 and r3 to be released?

Reply:
3. `v.21.7+rke2r2` is set as default on L3, could we bump that too? cc @rancher-max this was recently introduced so might need documentation to update cc @Oats87

Reply:
1. Correct yeah there's a typo and it should be rke2r3 (https://github.com/rancher/rke2/releases/tag/v1.21.7-rc2%2Brke2r3)
2. We have this PR for both to enable rancher QA to do upgrade checks, but if we don't want to merge this and have them point to brad's fork instead, that would probably be okay? These are early rc's for rke2 releases v1.20.14, v1.21.8, and v1.22.5, scheduled to release on Monday December 20th.
3. Due to (2), what is there right now should probably stay as the default.

Reply:
Ah got it, didn't know they were needed for upgrade testing. We can merge to avoid testing from forks 👍🏼

Reply:
Typo fixed, thanks!